Scenes/Player/PlayerStats.tscn

```diff
@@ -3,3 +3,4 @@
 [ext_resource path="res://Scenes/Player/Stats.tscn" type="PackedScene" id=1]
 
 [node name="PlayerStats" instance=ExtResource( 1 )]
+max_health = 3
```

Comment:
You shouldnt need a PlayerStats scene. The player stats persist in game_state. So at the beginning of player script you can call var game_state = get_node("/root/GameState"). This has set_heath, set_max_health functions

Reply:
You dont need to fetch the `GameState` it's a global variable :D

Reply:
I was trying to do it without the playerstats.. wasnt working well
Can try again - maybe i did something wrong along the way

and yup one can just call GameState, that's what I had started trying